### PR TITLE
Attempt to turn on additional build tests

### PR DESCRIPTION
**Pull request by rhatdan**
Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
Replaces https://github.com/containers/libpod/pull/6340

**Comment by mheon** [comment]
LGTM, but some tests are red

File: pkg/domain/infra/abi/system_novalink.go [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+import (
+	"context"
+
+	"github.com/containers/libpod/pkg/domain/entities"
+	"github.com/pkg/errors"
+)
+
+func (ic *ContainerEngine) VarlinkService(_ context.Context, opts entities.ServiceOptions) error {
+	return errors.Errorf("valink is not supported")
```
Comment (TomSweeneyRedHat) [inline pkg/domain/infra/abi/system_novalink.go:13]:
```suggestion
	return errors.Errorf("varlink is not supported")
```

File: pkg/domain/infra/abi/system_varlink.go [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+		if err := service.RegisterInterface(i); err != nil {
+			return errors.Errorf("unable to register varlink interface %v", i)
+		}
+	}
+
+	// Run the varlink server at the given address
+	if err = service.Listen(opts.URI, opts.Timeout); err != nil {
+		switch err.(type) {
+		case varlink.ServiceTimeoutError:
+			logrus.Infof("varlink service expired (use --timeout to increase session time beyond %s ms, 0 means never timeout)", opts.Timeout.String())
```
Comment (TomSweeneyRedHat) [inline pkg/domain/infra/abi/system_varlink.go:42]: is it timeout or time here now?

File: docs/source/markdown/podman-image.1.md [hunk 3]
```diff
@@ -14,6 +14,7 @@ The image command allows you to manage images
 | Command  | Man Page                                        | Description                                                                 |
 | -------- | ----------------------------------------------- | --------------------------------------------------------------------------- |
 | build    | [podman-build(1)](podman-build.1.md)            | Build a container using a Dockerfile.                                       |
+| diff     | [podman-image-diff(1)](podman-image-diff.1.md)  | Inspect changes on an image's filesystem.                                   |
```
Comment (TomSweeneyRedHat) [inline docs/source/markdown/podman-image.1.md:17]: need this change in commands-demo.md too

**Comment by TomSweeneyRedHat** [comment]
Just curious as to why we're hiding signature-policy?

**Comment by mheon** [comment]
LGTM

File: docs/source/markdown/podman-image.1.md (hunk 3 above)
Comment (rhatdan) [reply docs/source/markdown/podman-image.1.md:17]: I will do this in a different PR.  I want to get this in, so Matt can do a release.

File: pkg/domain/infra/abi/system_varlink.go (hunk 2 above)
Comment (rhatdan) [reply pkg/domain/infra/abi/system_varlink.go:42]: I will do a separate PR>

**Comment by TomSweeneyRedHat** [comment]
LGTM

**Comment by QiWang19** [comment]
`success Cancelled after 18m — Task Summary` Does this need rebase to rerun?

**Comment by mheon** [comment]
Damn nice job @rhatdan - that's almost all of our CI reenabled.

**Pull request merged**